addons/mail/static/src/components/discuss/tests/discuss_inbox_tests.js

```diff
@@ -75,7 +75,7 @@ QUnit.test('reply: discard on pressing escape', async function (assert) {
         "should display a single message"
     );
     await afterNextRender(() =>
-        document.querySelector('.o_Message_commandReply').click()
+        document.querySelector('.o_MessageActionList_action.fa-reply').click()
```

Comment:
Don't use the icon class a selector, this is prone to error and hard to maintain. Keep a name for the reply button.

Reply:
I find it weird to have classes that only exist to write tests against. I agree that this is not ideal though. Wouldn't it make more sense to use a data-attribute instead? (eg `data-msg-action="reply"`)

Reply:
`class` is the standard attribute to describe an element and to allow styling it / finding it, I have no issue with having one even if only for the tests. It also describes the element for many other reasons: makes the code easier to read, allows community to customize it easily, etc.